<launch>

    <param name="acca_folder" type="str" value="/home/acca/catkin_ws/src/ACCA" />
    <param name="odometry_topic" type="str" value="/fake_odom" />
    <param name="save_file_name" type="str" value="deliveryA3.csv"></param>
    <param name="save_parking" type="bool" value="false" />

    <param name="parking_area" type="str" value="k_parking2.csv" />

    <!-- Sensors -->

        <!-- LiDAR -->
        <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" />
        <!-- <node name="velodyne_laserscan_node" pkg="velodyne_laserscan" type="velodyne_laserscan_node" /> -->

    <!-- Sensors end -->
        
    <!-- Odometry -->
    <node name="fake_odometry" pkg="odometry" type="fakeOdometry.py" output="screen" />

    <!-- Path Planner -->
    <node name="create_global_path" pkg="path_planner" type="create_global_path.py" output="screen" />

    <!-- Localization -->
    <include file="$(find hdl_localization)/launch/hdl_localization.launch">
        <arg name="map_file" value="$(find hdl_localization)/data/KCityMap.pcd" /> 
    </include>


    <!-- Model -->
    <include file="$(find erp42_vehicle_rviz)/launch/erp42_vehicle_rviz.launch">
        <arg name="rvizconfig" value="/home/acca/model.rviz" />
    </include>

    <!-- Test -->
    <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0.0 0.0 0.0 0.0 0.0 0 map odom 10" />

</launch>